locales/en.lua

```diff
@@ -23,7 +23,7 @@ local Translations = {
         start_work = 'Start Working',
         stop_working = '[E] Stop Working',
         stop_work = 'Stop Working',
-        grab_stall = '[~g~G~s~] Grab Stall',
+        grab_stall = '[G] Grab Stall',
         drop_stall = '[~g~G~s~] Release Stall',

```

Comment:
I am getting [~g~G~s~] when picking the stall up, please change this to just [G]

Reply:
Hmmm no, sorry, that was already there and it also changes the color for when you use DrawText3D. I just fixed the [G] for when you use qb-target. If you are a QBCore developer please decide the design path so you can make an accurate fix to that vs using DrawText3D. This contribution is to add missing key information for when using qb-target.

Reply:
I will take a further look into this, thank you.